src/App.tsx

```diff
@@ -1,156 +1,117 @@
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        <Header
+          query={query}
+          setQuery={setQuery}
+        />
+
+        {todos && (
+          <section className={cn("todoapp__main")} data-cy="TodoList">
+            {preparedTodos.map((todo: Todo) => {
+              return (
+                <div
+                  key={todo.id}
+                  className={cn('todo', {
+                    'completed': todo.completed,
+                  })}
+                >
+                  {/* This is a completed todo */}
+                  <label className="todo__status-label">
+                    <input
+                      type="checkbox"
+                      className="todo__status"
+                      checked={todo.completed}
+                      onChange={() => { }}
+                    />
+                  </label>
+
+                  <span className="todo__title">{todo.title}</span>
+
+                  {/* Remove button appears only on hover */}
+                  <button type="button" className="todo__remove">x</button>
+
+                  {/* overlay will cover the todo while it is being updated */}
+                  <div className="modal overlay">
+                    <div className="modal-background has-background-white-ter" />
+                    <div className="loader" />
+                  </div>
+                </div>
+              )
+            })}
+          </section>
+        )}
+
+        {todos && (
+          <Footer
+            filterType={filterType}
+            setFilterType={setFilterType}
+            quantity={todos.length}
+            // haveCompleted={todos.some((todo) => todo.completed === true)}
+          />
+        )}
       </div>
 
-      {/* Notification is shown in case of any error */}
-      {/* Add the 'hidden' class to hide the message smoothly */}
-      <div className="notification is-danger is-light has-text-weight-normal">
-        <button type="button" className="delete" />
-
-        {/* show only one message at a time */}
-        Unable to add a todo
-        <br />
-        Unable to delete a todo
-        <br />
-        Unable to update a todo
-      </div>
+      {isError && (
+        <>
+          {/* Notification is shown in case of any error */ }
+          {/* Add the 'hidden' class to hide the message smoothly */}
+          <div className="notification is-danger is-light has-text-weight-normal">
+            <button type="button" className="delete" />
+
+            {/* show only one message at a time */}
+            Unable to add a todo
+            <br />
+            Unable to delete a todo
+            <br />
+            Unable to update a todo
+          </div>
+        </>
```

Comment:
Please create separate component for Error Messages and complete these points
Error messages
In case of any error show the notification with an appropriate message at the bottom

- the notification can be closed with the close button (add the hidden class);
- automatically hide the notification after 3 seconds;
- also hide the notification before any next request;
- use a wrong todos URL to test the error;